include/unifex/config.hpp.in

```diff
@@ -125,6 +125,18 @@
 #  define UNIFEX_NO_RTTI 0
 #endif
 
+#if __cpp_exceptions >= 199711
+#  define UNIFEX_NO_EXCEPTIONS 0
+#  define UNIFEX_TRY try
+#  define UNIFEX_CATCH(...) catch (__VA_ARGS__)
+#  define UNIFEX_RETHROW() throw
+#else
+#  define UNIFEX_NO_EXCEPTIONS 1
+#  define UNIFEX_TRY
+#  define UNIFEX_CATCH(...) if constexpr (true) {} else
+#  define UNIFEX_RETHROW() ((void)0)
```

Comment:
Should `UNIFEX_RETHROW()` be redefined to be `__builtin_unreachable()` or similar?
Or is the fact that it should be inside the UNIFEX_CATCH() block mean that it will never be instantiated anyway?

Reply:
> Should `UNIFEX_RETHROW()` be redefined to be `__builtin_unreachable()` or similar?
> Or is the fact that it should be inside the UNIFEX_CATCH() block mean that it will never be instantiated anyway?

I can't imagine it matters in an `if constexpr (false)` branch.